main.py

```diff
@@ -10,6 +10,7 @@
 from jinja2 import Template
 import contextlib
 from datetime import datetime
+from pidtree_bcc import utils as u
```

Comment:
`utils` isn't a long word. Prefer just using the `utils` name over `u` to avoid ambiguity.

Reply:
me: complains about ambiguous naming in code
also me: `import utils as u`

Good call, will fix

Reply:
Fixed in 90143be5f4d3710dd3dab6e2f3e800c3f2cc9189